radio/src/gui/colorlcd/model_mixes.cpp

```diff
@@ -286,6 +286,14 @@ class MixLineButton : public CommonInputOrMixButton
     if (line.flightModes) {
       drawFlightModes(dc, line.flightModes, textColor);
     }
+
+    uint8_t delayslow = 0;
+    if (line.speedDown || line.speedUp) delayslow = 1;
+    if (line.delayUp || line.delayDown) delayslow += 2;
+    if (delayslow) {
+      BitmapBuffer *delayslowbmp[] = {mixerSetupSlowIcon, mixerSetupDelayIcon, mixerSetupDelaySlowIcon};
+      dc->drawMask(width() - 16, PAGE_LINE_HEIGHT + FIELD_PADDING_TOP, delayslowbmp[delayslow - 1], textColor);
```

Comment:
There looks to be enough space on the end of the first line on both the NV14 and TX16S screens for this to go there, how about using `FIELD_PADDING_TOP` instead of `PAGE_LINE_HEIGHT + FIELD_PADDING_TOP` and moving this up to line 270? Then this will be visible if flight modes/switches/curves aren't in use.

Reply:
AFAIK delay does not have any effect without switch.
So I'm not sure it makes sense to show the icon on the first line.

Reply:
Not so. It applies a delay to the input, and slowdown as well. Add another mix line, the source as say your elevator stick, and apply a delay only, and watch what happens in the channel monitor when you move your sticks full range, and hold them for longer than the delay period. Same for slow. In a mix, the switch governs the mixer line being active or not, not whether slow or delay are in effect.

Reply:
Thank you - good to know.
Anyway, I made the changes you have suggested.